styx-cli-unshaded/src/test/java/com/spotify/styx/cli/PrettyCliOutputTest.java

```diff
@@ -95,6 +95,8 @@ private static Backfill backfill(String description) {
         .triggerParameters(TriggerParameters.builder().env("FOO", "bar").build())
         .nextTrigger(Instant.parse("2017-01-01T00:00:00Z"))
         .schedule(Schedule.DAYS)
+        .created(Instant.parse("2019-01-01T00:00:00Z"))
+        .lastModified(Instant.parse("2019-06-01T00:00:00Z"))
```

Comment:
Same as for the `PlainCliOutput`. 

In contrast, the `JsonCliOutput` is already covered since there we dynamically print all the contents of the backfill object as json, and don't have to specify each field separately. But the behaviour is visible in the `JsonCliOutputTest`.